src/LoopCarry.cpp

```diff
@@ -491,8 +490,8 @@ class LoopCarryOverLoop : public IRMutator {
             }
 
             allocs.push_back({scratch,
-                        loads[0][0]->type.element_of(),
-                        (int)c.size() * loads[0][0]->type.lanes(),
+                        loads[c.front()][0]->type.element_of(),
```

Comment:
The complexity of a problem is inversely proportional to the fix when measured in keystrokes :-).